util/topsql/reporter/reporter.go

```diff
@@ -346,6 +450,12 @@ func (tsr *RemoteTopSQLReporter) takeDataAndSendToReportChan(collectedDataPtr *m
 	for _, evict := range evicted {
 		normalizedSQLMap.LoadAndDelete(string(evict.SQLDigest))
 		normalizedPlanMap.LoadAndDelete(string(evict.PlanDigest))
+		others = addEvictedDataPoints(others, evict)
```

Comment:
Because the calling convention is always `sorted = addEvictedDataPoints(sorted, unsorted)`, we can move the `sort` out of the function, and change the util function to `addSortedEvictedDataPoints()`, which avoids redundant sort for the already-sorted list.